src/golf/ui.c

```diff
@@ -1121,9 +1121,9 @@ static void _golf_ui_in_game_paused(float dt) {
     if (_golf_ui_button_name(layout, "pause_menu_resume_button")) {
         golf_game_resume();
     }
-    if (_golf_ui_button_name(layout, "pause_menu_retry_button")) {
-        _golf_ui_start_fade_out(false, false, 0, true);
-    }
+    //if (_golf_ui_button_name(layout, "pause_menu_retry_button")) {

```

Comment:
(Small nitpick) - Don't leave commented out code, it confuses future contributors :yum:

Reply:
Apparently I forgot this. Thanks for pointing it out.